backend/frontend/tests/test_card_start_button.py

```diff
@@ -56,7 +56,7 @@ def test_start_button_moves_project_card_to_ip_column(self):
 
```

Comment:
I got this failed test and a other similar failures after a few runs. Maybe it's related to not using playwright `expect` like you added in other buttons? I also came across [this issue](https://stackoverflow.com/questions/77259434/python-testing-assertions-unittest-playwright) which seems related and encourages the use of `expect`

[More on expect](https://playwright.dev/python/docs/writing-tests#first-test:~:text=Playwright%20includes%20assertions%20that%20will%20wait%20until%20the%20expected%20condition%20is%20met.%20Using%20these%20assertions%20allows%20making%20the%20tests%20non%2Dflaky%20and%20resilient) - I'm reading in between the lines at this point, but `expect` might be the way to go?

```sh
FAIL: test_start_button_moves_topic_card_to_ip_column (frontend.tests.test_card_start_button.TestCardStartButton)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ngoako/Desktop/code/umuzi/Tilde/backend/frontend/tests/test_card_start_button.py", line 51, in test_start_button_moves_topic_card_to_ip_column
    self.assertIn(topic_card_title, ip_column)
AssertionError: 'something awesome part 31' not found in ''

----------------------------------------------------------------------
Ran 38 tests in 132.532s

FAILED (failures=1)
Destroying test database for alias 'default'...
```